packages/firebase_data_connect/firebase_data_connect/lib/src/common/dataconnect_options.dart

```diff
@@ -21,9 +21,9 @@ class ConnectorConfig {
   /// String representation of connectorConfig
   String toJson() {
     return jsonEncode({
-      'location': location,
-      'connector': connector,
-      'serviceId': serviceId,
+      location: location,
+      connector: connector,
+      serviceId: serviceId,
```

Comment:
If the keys are strings, then you should revert this change I believe.

Reply:
Is it a best practice to have the keys as quotes?

Reply:
The problem is, the key would be the same value as the value. 

So if you didn't change, the result would be:
```dart
{
"usa":"usa"
}
```

Rather than what I think would be more appropriate:
```dart
{
"location": "usa"
}
```

Reply:
Ah, thanks for catching that!